tubular/tests/test_panel_backend.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+
+import mock
+
+from tubular.panel_backend_api import PanelBackendAPI
+
+class PanelBackendAPITests(TestCase):
+    """
+    Unit tests for Panel Backend.
+    """
+
+    def setUp(self):
+        """
+        Setup data for test cases.
+        """
+        self.base_url = 'https://example.com'
+        self.auth_token = 'token'
+
+
+    def test_retire_user(self):
+        """
+        Test that api endpoint.
+        """
+        with mock.patch('tubular.panel_backend_api.requests.post') as mock_post:
+            mock_post.return_value.status_code = 200
+            response = PanelBackendAPI.retire_user(self, learner={'original_username': 'test'})
+
+        assert response.status_code == 200
```

Comment:
should be a test case for invalid response too

Reply:
done